src/libsystemd/sd-bus/sd-bus.c

```diff
@@ -2604,6 +2604,21 @@ _public_ int sd_bus_get_timeout(sd_bus *bus, uint64_t *timeout_usec) {
         }
 }
 
+int sd_bus_reply(sd_bus_message *call, sd_bus_message *reply) {
```

Comment:
Shouldn't `call` should be `const`? We're not modify that object, only the bus which it refers to.